pkg/cmd/cli/describe/describer.go

```diff
@@ -600,11 +603,45 @@ func (d *ImageStreamDescriber) Describe(namespace, name string) (string, error)
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, imageStream.ObjectMeta)
 		formatString(out, "Docker Pull Spec", imageStream.Status.DockerImageRepository)
+		describeImageStreamQuota(out, d.OSClient, d.KubeClient, imageStream)
 		formatImageStreamTags(out, imageStream)
 		return nil
 	})
 }
 
+func describeImageStreamQuota(out *tabwriter.Writer, c client.Interface, kc kclient.Interface, stream *imageapi.ImageStream) {
+	quotas, err := kc.ResourceQuotas(stream.Namespace).List(kapi.ListOptions{})
+	if err != nil {
+		return
+	}
+
+	var limit resource.Quantity
+	for _, item := range quotas.Items {
+		// search for smallest ImageStream quota
+		if value, ok := item.Spec.Hard[imageapi.ResourceImageStreamSize]; ok {
+			if limit.Value() == 0 || limit.Cmp(value) > 0 {
+				limit = value
+			}
+		}
+	}
+	if limit.Value() != 0 {
+		quantity := imagequota.GetImageStreamSize(c, stream, make(map[string]*imageapi.Image))
+		// we use manual scaling here, to make sure we print correct GiB/MiB values
+		// independently of what format was used for specifying limit
+		const gib = 1 << 30
+		const mib = 1 << 20
+		if quantity.Value() >= gib || limit.Value() >= gib {
+			fq := float64(quantity.Value()) / float64(gib)
+			fl := float64(limit.Value()) / float64(gib)
+			formatString(out, "Quota Usage", fmt.Sprintf("%.2fGiB / %.2fGiB", fq, fl))
+		} else {
+			fq := float64(quantity.Value()) / float64(mib)
+			fl := float64(limit.Value()) / float64(mib)
+			formatString(out, "Quota Usage", fmt.Sprintf("%.0fMiB / %.0fMiB", fq, fl))
```

Comment:
> Limiting the output to just 3 digits is a bit too space-saving. Can we have also precision of 2?

At this level (MiB) being accurate to any digits after comma isn't necessary, imho. 1MiB precision is sufficient. 

 I'll update the floating point math.